app/components/Views/confirmations/components/TransactionReview/TransactionReviewInformation/index.js

```diff
@@ -300,9 +304,10 @@ class TransactionReviewInformation extends PureComponent {
     } catch (error) {
       Logger.error(error, 'Navigation: Error when navigating to buy ETH.');
     }
-    InteractionManager.runAfterInteractions(() => {
-      Analytics.trackEvent(MetaMetricsEvents.RECEIVE_OPTIONS_PAYMENT_REQUEST);
-    });
+
+    this.props.metrics.trackEvent(
```

Comment:
Destructure metrics on L299
```suggestion
    metrics.trackEvent(
```